python/tests/conftest.py

```diff
@@ -0,0 +1,52 @@
+import os
+import typing
+
+import pymongo
+import pytest
+from energuide import database
+
+@pytest.fixture
+def username() -> str:
+    return os.environ.get(database.EnvVariables.username.value, database.EnvDefaults.username.value)
+
+@pytest.fixture
```

Comment:
There's also a PEP rule about 2 lines between module-level functions/classes. It's not in Pylint for some reason, but it drives my editor nuts.